SpatialGDK/Source/SpatialGDK/Public/Utils/RPCContainer.h

```diff
@@ -27,11 +27,11 @@ class FRPCContainer
 public:
 	void QueueRPC(FPendingRPCParamsPtr Params, ESchemaComponentType Type);
 	void ProcessRPCs(const FProcessRPCDelegate& FunctionToApply);
-	bool ObjectHasRPCsQueuedOfType(const FUnrealObjectRef& TargetObjectRef, ESchemaComponentType Type) const;
+	bool ObjectHasRPCsQueuedOfType(const Worker_EntityId& EntityId, ESchemaComponentType Type) const;
```

Comment:
```suggestion
	bool ObjectHasRPCsQueuedOfType(Worker_EntityId EntityId, ESchemaComponentType Type) const;
```
A matter of preference